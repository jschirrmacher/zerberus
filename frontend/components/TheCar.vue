<script setup lang="ts">
import CarWheel from "./CarWheel.vue"
import CarChassis from "./CarChassis.vue"
</script>

<template>
  <div class="car">
    <!-- <CarWheel /> -->
    <!-- <CarWheel /> -->
    <CarChassis />
    <!-- <CarWheel />
    <CarWheel /> -->
  </div>
</template>

<style scoped lang="scss"></style>
