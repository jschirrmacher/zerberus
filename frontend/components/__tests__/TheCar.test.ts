import { describe, it, expect } from "vitest"
import { shallowMount } from "@vue/test-utils"
import TheCar from "../TheCar.vue"

describe("TheCar", () => {
  it("should render as expected", () => {
    const component = shallowMount(TheCar)
    expect(component.element).toMatchSnapshot()
  })
})
