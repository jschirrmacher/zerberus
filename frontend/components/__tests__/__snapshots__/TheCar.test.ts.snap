// Vitest Snapshot v1

exports[`TheCar > should render as expected 1`] = `
<div
  class="car"
>
  <!-- &lt;CarWheel /&gt; -->
  <!-- &lt;CarWheel /&gt; -->
  <car-chassis-stub />
  <!-- &lt;CarWheel /&gt;
    &lt;CarWheel /&gt; -->
</div>
`;
